src/management-system-v2/app/(dashboard)/[environmentId]/processes/[processId]/wrapper.tsx

```diff
@@ -232,10 +239,12 @@ const Wrapper = ({ children, processName, processes }: WrapperProps) => {
     }
   };
 
+  const childrenArray = Children.toArray(children);
```

Comment:
there might be better ways to handle this but an array seemed to be the most shallow change